model/contract/UsageControlComponent.ttl

```diff
@@ -0,0 +1,44 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix ids: <https://w3id.org/idsa/core/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+# Classes
+# -------
+
+ids:UsageControlComponent
+    a owl:Class;
+    rdfs:label "Usage Control Component"@en ;
+    rdfs:comment "Defined usage control component (i.e. PIP, PXP, etc.) in the IDS."@en;
+	.
+
+ids:PXP
+    a owl:Class;
+    rdfs:subClassOf ids:UsageControlComponent;
+    rdfs:label "policy execution point"@en;
+    rdfs:comment "The reference to the PXP that operates a specific action, URI or reference to an associated PXP interface."@en;
+.
+
+ids:PIP
+    a owl:class;
```

Comment:
Must be `owl:Class`. @HaydarAk do we actually have an automated check for such issues of ontology validity in place? We should!

Reply:
I corrected it.